dir.props

```diff
@@ -114,7 +114,7 @@
   <!-- Dependency auto-upgrade configuration -->
   <ItemGroup>
     <DependencyBuildInfo Include="CoreFx">
-      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/$(DependencyBranch)</RawUrl>
+      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/dev/api</RawUrl>
```

Comment:
I wonder what the best way is to make sure this doesn't break master when we merge this branch into master.